layers/+os/osx/keybindings.el

```diff
@@ -23,12 +23,12 @@
   ;; `Alist' linking the layer config variables to
   ;; the internal Emacs variables for the modifier keys.
   (setq modifier-keys '((osx-command-as       . mac-command-modifier)
-                        (osx-option-as        . mac-option-modifier)
-                        (osx-function-as      . mac-function-modifier)
-                        (osx-control-as       . mac-control-modifier)
-                        (osx-right-command-as . mac-right-command-modifier)
-                        (osx-right-option-as  . mac-right-option-modifier)
-                        (osx-right-control-as . mac-right-control-modifier)))
+                         (osx-option-as        . mac-option-modifier)
```

Comment:
The indentation for these lines should probably be restored to default.
Otherwise they will appear in a future unrelated commit if someone for example indents the whole buffer by pressing: `=ig` (with the evil editing style).

Reply:
i think the pared it gives me this indent. Fixed